locale/circuitpython.pot

```diff
@@ -4043,6 +4061,10 @@ msgstr ""
 msgid "start/end indices"
 msgstr ""
 
+#: ports/raspberrypi/bindings/tca9555r/__init__.c
+msgid "state only supports 16 bits"
+msgstr ""
+
```

Comment:
Instead, use `mp_arg_validate_int_range`

Reply:
Where possible, you should use the routines defined in `py/argcheck.c`. (declaration in `py/runtime.h`) This helps reduce code size and makes error messages more consistent.

For instance, here's the implementation of one of them:
```
mp_int_t mp_arg_validate_int_range(mp_int_t i, mp_int_t min, mp_int_t max, qstr arg_name) {
    if (i < min || i > max) {
        mp_raise_ValueError_varg(translate("%q must be %d-%d"), arg_name, min, max); 
    }
    return i;
}   
```
you'd call `mp_arg_validate_int_range(state, 0, UINT16_MAX, MP_QSTR_state)` instead of `if(...) { mp_raise_TypeError...  }`.

Similar for the type check, `mp_arg_validate_type(pin_obj, &tca_pin_type, MP_QSTR_pin)`.

Reply:
Switched over to argchecks, and even added a new one for the dual pin type checking both `tca` and `cyw` do.